pillar.example

```diff
@@ -3,6 +3,8 @@ resolver:
   ng:
     resolvconf:
       enabled: False
+    networkmanager:
```

Comment:
Could we have a `enabled` flag in case someone wanted to disable it?

```
networkmanager:
  enabled: True
```

and then we can have if statement inside the state.

Reply:
Good idea - done!!